extensions/wikia/Helios/HelperController.class.php

```diff
@@ -118,8 +120,28 @@ public function sendConfirmationEmail()
 			return;
 		}
 
-		$mailTemplate = $this->app->renderView( 'UserLogin', 'GeneralMail', [ 'language' => $user->getOption( 'language' ), 'type' => 'confirmation-email' ] );
-		$mailStatus = $user->sendConfirmationMail( false, 'ConfirmationMail', 'usersignup-confirmation-email', true, $mailTemplate );
+		$langCode = $this->getVal( 'langCode', 'en' );
+		$mailTemplate = $this->app->renderView(
+			'UserLogin',
+			'GeneralMail',
+			[
+				'language' => $langCode,
+				'type' => 'confirmation-email'
+			]
+		);
+
+		$lang = \Language::factory($langCode);
+		$mailStatus = (new GlobalStateWrapper(['wgLang' => $lang]))
+			->wrap(function() use ($user, $mailTemplate, $langCode) {
+				return $user->sendConfirmationMail(
+					false,
+					'ConfirmationMail',
+					'usersignup-confirmation-email',
+					true,
+					$mailTemplate,
+					$langCode
```

Comment:
Oh, I see it [here](https://github.com/Wikia/app/pull/7346/files#diff-897fba1dd6b8edc3238762e35d0b7c71R3633). Does `$wgLang` also need to be wrapped?